collector/lib/runtime-control/Config.h

```diff
@@ -8,23 +8,47 @@
 
 #include <storage/runtime_filters.pb.h>
 
+#include "Hash.h"
+
+#include "../ResourceSelector.h"
+
 namespace collector::runtime_control {
 
 class Config {
  public:
+  Config() {
+    for (int i = 0; i < storage::RuntimeFilter_RuntimeFilterFeatures_RuntimeFilterFeatures_ARRAYSIZE; i++) {
+      kFeatureFlags_[i] = 1UL << (i + 1);
+    }
+  }
+
   static Config& GetOrCreate();
 
   // returns true when initialized, false for a timeout.
   bool WaitUntilInitialized(unsigned int timeout_ms);
 
   void Update(const storage::RuntimeFilteringConfiguration& msg);
+  bool IsFeatureEnabled(uint64_t bitMask, storage::RuntimeFilter_RuntimeFilterFeatures feature);
+  bool IsFeatureEnabled(std::string cluster, std::string ns, std::string container_id, storage::RuntimeFilter_RuntimeFilterFeatures feature);
```

Comment:
In the final version of this API, the return value should make it possible to return an `unknown` state for when the container->namespace mapping is not (yet) available.
`std::optional<bool>` maybe ?